ambari-server/src/main/resources/common-services/AMBARI_METRICS/3.0.0/package/scripts/metrics_collector.py

```diff
@@ -22,21 +22,21 @@
 from resource_management.libraries.functions.security_commons import build_expectations, \
   cached_kinit_executor, get_params_from_filesystem, validate_security_config_properties, \
   FILE_TYPE_XML
-from ams import ams
-from ams_service import ams_service
-from hbase import hbase
-from status import check_service_status
+from scripts.ams import ams
```

Comment:
Ambari metrics collector start is failing with following error
`Traceback (most recent call last):
  File "/var/lib/ambari-agent/cache/common-services/AMBARI_METRICS/3.0.0/package/scripts/metrics_collector.py", line 25, in <module>
    from scripts.ams import ams
ModuleNotFoundError: No module named 'scripts'